icepyx/tests/test_auth.py

```diff
@@ -1,10 +1,15 @@
+import os
+
 import pytest
 import requests
 
 import earthaccess
 
 from icepyx.core.auth import EarthdataAuthMixin
 
+# Are we running on Travis CI?
+TRAVIS = os.environ.get("TRAVIS") == "true"
```

Comment:
What do you think of:

```python
skip_if_travis_ci = pytest.mark.skipif(TRAVIS, reason="Skipping this test on Travis CI.")


@skip_if_travis_ci
def test_something:
    ...
```

Reply:
I have never seen that before.  Does it work?

Reply:
```suggestion
skip_if_travis_ci = pytest.mark.skipif(
    os.environ.get("TRAVIS") == "true", reason="Skipping this test on Travis CI."
)
```